rasa/shared/core/generator.py

```diff
@@ -217,6 +217,40 @@ def update(self, event: Event, skip_states: bool = False) -> None:
 
             self._append_current_state()
 
+    def fingerprint(self) -> Text:
+        """Returns a unique hash for the tracker which is stable across python runs.
+
+        Returns:
+            fingerprint of the tracker
+        """
+        data = []
+        sender_id_hash = rasa.shared.utils.io.get_text_hash(self.sender_id)
+        data.append(sender_id_hash)
+
+        if self.domain:
+            data.append(self.domain.fingerprint())
```

Comment:
@twerkmeister added functionality for `.fingerprint` to `deep_container_fingerprint` in https://github.com/RasaHQ/rasa/pull/9175.

Reply:
I can see the PR hasn't been merged it so I can't use this added functionality at the moment. 
Would it be fine to actually change this to:
```
data.append(self.domain.as_dict())
```

Reply:
it's only the 4 lines in this file: https://github.com/RasaHQ/rasa/pull/9175/files#diff-d0c59274cb6aad666a5addf9f758510ba9a2ecdcb48b08a16b904f90c980e017R221

So probably easier to just copy the changes across to this branch.